Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -673,6 +674,17 @@ - (void)updateServiceWithProfileName:(nullable NSString *)localProfileName
     });
 }
 
+- (void)syncLocalProfile
```

Comment:
nit: `fetchLocalUsersProfile` or `pullLocalProfile` might better indicate the direction of data flow here, vs sync which could be either push or pull.

Reply:
Good point!